package.json

```diff
@@ -21,10 +21,15 @@
     "storybook": "start-storybook -p 6007",
     "build-storybook": "build-storybook"
   },
-  "keywords": [],
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/poetapp/explorer-web.git"
+  },
   "license": "MIT",
-  "greenkeeper": {
+  "bugs": {
+    "url": "https://github.com/poetapp/explorer-web/issues"
+  },
+   "greenkeeper": {
```

Comment:
seems to be an extra space here before greenkeeper.

Reply:
Ah, OK.

Reply:
Was already closed so I created a new PR.